web/pdf_page_view.js

```diff
@@ -362,6 +366,7 @@ class PDFPageView extends PDFPageViewBase {
     this.#annotationMode =
       options.annotationMode ?? AnnotationMode.ENABLE_FORMS;
     this.imageResourcesPath = options.imageResourcesPath || "";
+    this.enableDetailCanvas = options.enableDetailCanvas || false;
```

Comment:
The default value should agree with `AppOptions`.
`this.enableDetailCanvas = options.enableDetailCanvas !== false;`